R/geom-violin.r

```diff
@@ -75,7 +81,7 @@ geom_violin <- function(mapping = NULL, data = NULL, stat = "ydensity",
       trim = trim,
       scale = scale
     ),
-    params = list(...)
+    params = list(draw_quantiles = draw_quantiles, ...)
```

Comment:
This should go in `stat_params`, not params, because you know that it belongs to the stat (and not the geom)